.github/workflows/ci.yml

```diff
@@ -19,22 +20,30 @@ jobs:
 
   lint:
     name: Lint
-    runs-on: windows-2019
+    runs-on: windows-2022
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: true
       - run: rustup toolchain install stable --profile minimal --component clippy --no-self-update
+      - name: Install LLVM and Clang
+        uses: KyleMayes/install-llvm-action@v2
+        with:
+          version: ${{ env.LLVM_VERSION }}
       - uses: Swatinem/rust-cache@v2
       - run: cargo clippy --all-features -- -D warnings
 
   test:
     name: Test
-    runs-on: windows-2019
+    runs-on: windows-2022
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: true
       - run: rustup toolchain install stable --profile minimal --no-self-update
+      - name: Install LLVM and Clang
+        uses: KyleMayes/install-llvm-action@v2
+        with:
+          version: ${{ env.LLVM_VERSION }}
```

Comment:
The runner should have the latest version of LLVM, 20.1.7. Why do you install it, because it can be downgraded in the future.

Also, a nitpick, I prefer to keep it simple with a step like the following instead of  using a thrid party action:
```powershell
      - name: Pin LLVM to a specific version
        run: |
          $latestChocoVersion = (Resolve-ChocoPackageVersion -PackageName "llvm" -TargetVersion $env:LLVM_VERSION)
          Install-ChocoPackage -PackageName llvm -ArgumentList '--allow-downgrade', '--version', $latestChocoVersion
```

Reply:
done in [1e7ecc8](https://github.com/jac3km4/red4ext-rs/pull/74/commits/1e7ecc8fcff3ea9e1f5f7ae71618707cb7d2f26d) and 
173f778

Reply:
Is it necessary to pin the LLVM version? I mean, you could use the one in the runner already.

Reply:
RED4ext is pinned because of `clang-format`. Sometimes they add new options that change the formatting and Visual Studio uses an older version of `clang-format` than the one installed on the runner.